data/tlg0090/tlg001/tlg0090.tlg001.opp-grc1.xml

```diff
@@ -0,0 +1,451 @@
+                            Στηλῶν διὰ Σαρδοῦς, Σικελίας, Πελοποννήσου, (Ἰωνίας,) Καρίας, Λυκίας,
+                            Παμφυλίας, Κιλικίας, καὶ Ταύρου ἑξῆς ἕως Ἰμάου ὄρους. τῶν τοίνυν τόπων
+                            τὸ μὲν βόρειον, τὸ δὲ νότιον ὀνομάζει. </p>
+                    </div>
+                </div>
+
+             
+                <div type="textpart" subtype="chapter" n="2">
+                    <div type="textpart" subtype="section" n="6">
+                           <head> [Περὶ ἀνέμων] </head>
+                        <p> 
+                            Ἄνεμοι δὲ πνέουσιν ἀπὸ μὲν ἰσημερινῆς ἀνατολῆς ἀπηλιώτης, ἀπὸ
+                            δὲ ἰσημερινῆς δύσεως ζέφυρος, ἀπό δὲ μεσημβρίας νότος, ἀπὸ δὲ ἄρκτου
+                            ἀπαρκτίας, ἀπὸ δὲ τροπῆς θερινῆς καικίας, ἑξῆς δὲ ἀπ᾿ ἰσημερινῆς
+                            ἀνατολῆς ἀπηλιώτης, καὶ ἀπὸ χειμερινῆς εὖρος· δυσμικοὶ δὲ ἀπὸ μὲν
+                            δύσεως χειμερινῆς Λίψ, καὶ ἑξῆς πάλιν ἀπὸ δύσεως ἰσημερινῆς
+                            ζέφυρος, ἀπὸ δὲ δύσεως θερινῆς ἀργέστης ἤτοι Ὀλυμπίας· εἶτα νότος 
+                            καὶ ἀπαρκτίας ἀντιπνέοντες. Γίνονται
+                            οὖν ὀκτώ. 
+                        </p>
+                    </div>
+              
+                    <div type="textpart" subtype="section" n="7">
+                        <p> 
+                            Τιμοσθένης δὲ, ὁ γράψας τοὺς περίπλους, δώδεκά φησι,
+                            προστιθεὶς μέσον ἀπαρκτίου καὶ καικίου βορέαν, εὔρου δὲ καὶ νότου
+                            Φοίνικα τὸν καὶ εὐρόνοτον, μέσον δὲ νότου καὶ Λιβὸς τὸν λευκόνοτον
+                            ἤτοι Λιβόνοτον, μέσον δὲ ἀπαρκτίου καὶ ἀργέστου Θρασκίαν ἤτοι
+                            κίρκιον ὑπὸ τῶν περιοίκων [ὀνομαζόμενον]. Ἔθνη δὲ οἰκεῖν τὰ πέρατα·
+                            κατ᾿ ἀπηλιώτην Βακτριανοὺς, κατ᾿ εὖρον Ἰνδοὺς, κατὰ Φοίνικα
+                            Ἐρυθρὰν θάλασσαν καὶ Αἰθιοπίαν, κατὰ λευκόνοτον τοὺς ὑπὲρ Σύρτεις Γαράμαντας, κατὰ
+                            Λίβα Αἰθίοπας δυσμικοὺς [τοὺς] ὑπὲρ Μαύρους, κατὰ ζέφυρον Στήλας
+                            καὶ ἀρχὰς Λιβύης καὶ Εὐρώπης, κατ᾿ ἀργέστην Ἰβηρίαν τὴν νῦν
+                            Ἱσπανίαν, κατὰ δὲ Θρασκίαν [Κελτοὺς καὶ τὰ ὅμορα, κατὰ δ᾿ ἀπαρκτίαν] 
+                            τοὺς ὑπὲρ Θρᾴκην Σκύθας, κατὰ βορρᾶν Πόντον, Μαιῶτιν,
+                            Σαρμάτας· κατὰ καικίαν Κασπίαν θάλασσαν καὶ Σάκας.
+                        </p>
+                    </div>
+                </div>
+
+               
+                <div type="textpart" subtype="chapter" n="3">
+                    <div type="textpart" subtype="section" n="8">
+                         <head>[Περίπλους θαλασσῶν]                   
+</head>
+                        <p> 
+                            Ἔστι δὲ ἡ μεγάλη Σύρτις σταδίων &#885;ε, ἡ δὲ μικρὰ σταδίων &#885;αχ&#884;, τὸ δὲ
```

Comment:
Avoid use of xml entities : http://unicode.org/cldr/utility/character.jsp?a=0375